src/NServiceBus.AcceptanceTests/Serialization/When_registering_custom_serializer.cs

```diff
@@ -91,19 +92,21 @@ public void Serialize(object message, Stream stream)
                 serializer.Serialize(stream, message);
             }
 
-            public object[] Deserialize(Stream stream, IList<Type> messageTypes = null)
+            public object[] Deserialize(ReadOnlyMemory<byte> body, IList<Type> messageTypes = null)
             {
-                var serializer = new System.Xml.Serialization.XmlSerializer(typeof(MyRequest));
+                using (var stream = new MemoryStream(body.ToArray()))
```

Comment:
MemoryStream does not need to be disposed.

Reply:
I don't think it's worth tweaking the tests